src/ls.c

```diff
@@ -260,6 +260,21 @@ SDB_API void *ls_pop(SdbList *list) {
 	return NULL;
 }
 
+SDB_API SdbList *ls_clone(SdbList *list) {
```

Comment:
oh. this may conflict after merging the other pr

Reply:
It's the same commit, will disappear on rebase.